docs/rector_rules_overview.md

```diff
@@ -4861,7 +4861,7 @@ Remove null coalescing operator ??=
 ### `DowngradeTypedPropertyRector`
 
 - class: [`Rector\DowngradePhp74\Rector\Property\DowngradeTypedPropertyRector`](/rules/downgrade-php74/src/Rector/Property/DowngradeTypedPropertyRector.php)
-- [test fixtures](/rules/downgrade-php74/tests/Rector/Property/DowngradeTypedPropertyRector/Fixture)
+- [test fixtures](/rules/downgrade-php74/tests/Rector/Property/NoDocBlockDowngradeTypedPropertyRector/Fixture)
```

Comment:
@TomasVotruba I don't know why is this still happening.

Reply:
I think it's a path matching bug in `dump-rectors` command.

The end path of both files is the same, so the slash need to be added to regular expression or path matching or whatever is used there to find a test case

Reply:
I can, it's order issue.

Reply:
Here: #4260